README.md

```diff
@@ -12,7 +12,7 @@ The best feature is Lottie animations in refresh view, it uses lottie animations
 - Customize the animation view by providing your own custom RefreshView (need to subclass SSAnimationView)
 - Set Gif animations in refresh view
 - Import lottie jason in assets folder and apply animation ( as simple as that )
```

Comment:
Typo: json file instead of Jason. Also it's not necessary to keep space between starting and ending brackets. As in line no. 12 there is no space (need to subclass SSAnimationView). Check other places as well.